janus_core/calculations/geom_opt.py

```diff
@@ -93,6 +101,13 @@ def optimize(  # pylint: disable=too-many-arguments,too-many-locals,too-many-bra
     log_kwargs.setdefault("name", __name__)
     logger = config_logger(**log_kwargs)
 
+    s_grp = spacegroup(struct, symmetry_precision, angle_precision)
```

Comment:
Does this actually do anything with the space-group beyond print it? Does it allow constraints?

Reply:
yes and not... for the moment only printing you can constrain symmetry in ase but you need to know it.

Reply:
spglib can return sets of symops. Perhaps for a future PR though